src/security/authenticated_user.js

```diff
@@ -51,7 +51,7 @@ var AuthenticatedUser = mCAP.User.extend({
 
     if (attr.preferences && !(attr.preferences instanceof mCAP.Model) && this.get('preferences')) {
```

Comment:
I suggest a helper function `attr.preferences = removeEmptyStringValues(attr.preferences)`, but maybe this is too much